### ISSUE TITLE
mdl-button-raised to be fusionned with mdl-button

### ISSUE THREAD

**Issue by Piou2fois** [jadjoubran/angular-material-design-lite#10]
I think it would be better to have an attribute for the "design" to include :
-raised
-fab
-mini-fab
-icon
-colored
-primary
-accent
and why not ripple
Everything could be into a single attribute in which you can put a list 
<mdl-button design="raised fab icon primary">Click me</mdl-button>

**Comment by jadjoubran** [comment]
Yes makes sense, except for the ripple because of the configProvider.. Here's the logic behind it:

Since people need consistency in their apps, they're going to stick with either with ripples or without ripples.. This is why I abstracted it into the configProvider (same thing for the text-field floating)

So it would make sense to have a `raised` attribute, a `theme` for the accent, primary and default (this has already been implemented)

And I'll start looking into the other options to include the fab, mini-fab and icon

**Comment by jadjoubran** [comment]
Also the removal of `mdl-button-raised` is backwards incompatible, so I'll probably keep it (and issue a warning) for 2 versions before completely removing it.

**Comment by Piou2fois** [comment]
Idea could be to put a "console.log('warning, mdl-button-raised will be unsupported soon') in the directive ?

**Comment by jadjoubran** [comment]
yep
